Packs/Armis/Integrations/Armis/Armis.py

```diff
@@ -0,0 +1,669 @@
+                'dataSources',
+                'category',
+                'type',
+                'model',
+                'manufacturer',
+                'macAddress',
+                'ipAddress',
+                'site',
+                'tags',
+                'operatingSystem',
+                'operatingSystemVersion',
+                'sensor',
+                'user',
+                'accessSwitch',
+                'firstSeen',
+                'id',
+                'ipv6',
+                'lastSeen',
+                'visibility',
+            ], removeNull=True)
+        )
+    return 'No devices found'
+
+
+def search_devices_by_aql_command(client: Client, args):
+    """
+    armis-search-devices-by-aql command: Returns results for searching Devices using a free AQL string
+    Args:
+        client (Client): An Armis client object
+        args (dict): A dict object containing the arguments for this command
+    """
+    aql_string = args.get('aql_string')
+    max_results = int(args.get('max_results', 50))
+
+    response = client.free_string_search_devices(aql_string, max_results=max_results)
+    results = response.get('results')
+    if results:
+        return CommandResults(
+            outputs_prefix='Armis.SearchDevicesByAql',
+            outputs_key_field='deviceId',
+            outputs=results,
+            raw_response=response,
+            readable_output=tableToMarkdown('Devices', results, headers=[
+                'riskLevel',
+                'name',
+                'dataSources',
+                'category',
+                'type',
+                'model',
+                'manufacturer',
+                'macAddress',
+                'ipAddress',
+                'site',
+                'tags',
+                'operatingSystem',
+                'operatingSystemVersion',
+                'sensor',
+                'user',
+                'accessSwitch',
+                'firstSeen',
+                'id',
+                'ipv6',
+                'lastSeen',
+                'visibility',
+            ], removeNull=True)
+        )
+    return 'No devices found'
+
+
+def search_alerts_by_aql_command(client: Client, args):
+    """
+   armis-search-alerts-by-aql command: Returns results for searching Alerts using a free AQL string
+    Args:
+        client (Client): An Armis client object
+        args (dict): A dict object containing the arguments for this command
+    """
+    aql_string = args.get('aql_string')
+    max_results = int(args.get('max_results', 50))
+
+    response = client.free_string_search_alerts(aql_string, max_results=max_results)
+    results = response.get('results')
+    if results:
+        return CommandResults(
+            outputs_prefix='Armis.SearchAlertsByAql',
+            outputs_key_field='alertId',
+            outputs=results,
+            raw_response=response,
+            readable_output=tableToMarkdown('Alerts', results, headers=[

```

Comment:
The headers appear according to their order here, so lets have it more organized:
[id, description, type, title, severity, status ..... (the rest doesn't really matter)